app/posts/register-trainee-teachers/register-trainee-teachers.md

```diff
@@ -0,0 +1,23 @@
+---
+tags: false
+layout: collection
+title: Register trainee teachers
+description: A service for collecting and maintaing trainee teacher registration data
+# related:
+#   items:
+#   - text: Prototype
+#     href: https://manage-courses-prototype.herokuapp.com
```

Comment:
Does @edwardhorsford have a public link for this yet?

Reply:
Patience @paulrobertlloyd!